### ISSUE TITLE
different orders / cuts

### ISSUE THREAD

**Issue by brakthehun** [A-Metaphysical-Drama/J-Tech-Photonics-Laser-Tool#1]
It would be nice to have an option for order of cuts by colors or have some cuts be different speed to engrave some and cut others

**Comment by A-Metaphysical-Drama** [comment]
Yeah that would be a nice feature, i should update this addon, i guess also the original authors made some updates in the meantime
